dts/arm/nordic/nrf52840.dtsi

```diff
@@ -174,6 +174,22 @@
 			interrupt-names = "wdt";
 			label = "WDT";
 		};
+
+		cryptocell: crypto@5002a000 {
```

Comment:
a --> A

Reply:
I can't do that, because I am getting following warning:

```
nrf52840_pca10056.dts_compiled: Warning (simple_bus_reg): Node /soc/crypto@5002A000 simple-bus unit address format error, expected "5002a000"
```

Reply:
Looks like it is only for entries inside `soc`. That's why `flash-controller` have no problem, but if you look at `spi3: spi@4002B000`, which is inside `soc`:

```
nrf52840_pca10056.dts_compiled: Warning (simple_bus_reg): Node /soc/spi@4002B000 simple-bus unit address format error, expected "4002b000"
```

Reply:
I don't get it. SPI3 is already with capital "B".